api/src/controllers/referent.js

```diff
@@ -564,6 +564,19 @@ router.get("/:id", passport.authenticate("referent", { session: false }), async
   }
 });
 
+router.get("/", passport.authenticate(["referent"], { session: false }), async (req, res) => {
+  try {
+    const email = req.query.email;
+    console.log("email", email);
+    let data = await ReferentModel.findOne({ email });
+    console.log("Data", data);
+    return res.status(200).send({ ok: true, data });
```

Comment:
on essaie de serialiser (reformater) les infos qu'on renvoit (supprimer les tokens et autres infos à ne pas partager)

```suggestion
    return res.status(200).send({ ok: true, data: serializeReferent(referent, req.user) });
```